cmd/cat-main.go

```diff
@@ -154,9 +154,6 @@ func parseCatSyntax(ctx *cli.Context) (args []string, versionID string, timeRef
 		fatalIf(errInvalidArgument().Trace(), "You need to pass at least one argument if --version-id is specified")
 	}
 
-	if len(args) == 0 {
```

Comment:
This is a functional change, right?

Why is it needed again?

Reply:
> This is a functional change, right?
> 
> Why is it needed again?

This is a regression for an earlier change, this is not needed.

Reply:
ok